dm/dm/worker/config.go

```diff
@@ -99,6 +99,8 @@ type Config struct {
 	KeepAliveTTL      int64 `toml:"keepalive-ttl" json:"keepalive-ttl"`
 	RelayKeepAliveTTL int64 `toml:"relay-keepalive-ttl" json:"relay-keepalive-ttl"`
 
+	RelayDir string `toml:"relay-dir" json:"relay-dir"`
```

Comment:
seems this change need PM's review @sunzhaoyang

Reply:
btw why we need this extra config?

Reply:
the storage path of relay log is more related to the property of DM-worker, such as disk usage

Reply:
LGTM